<launch>

    <!-- gazebo server -->
    <node pkg="pioneer3at" type="bashim" name="gazebo_server" args="gzserver $(find pioneer3at)/config/gazebo/teste.sdf" />


    <param name="robot_description" command="$(find xacro)/xacro.py $(find pioneer3at)/urdf/pioneer3at2.xacro" />
    <!--<param name="Pioneer3AT/urdf" command="$(find xacro)/xacro.py $(find pioneer3at)/urdf/pioneer3at2.xacro" />-->

    <group ns="robot1">
       <include file="$(find pioneer3at)/launch/explorer_robot1.launch"/>
    </group>

    <group ns="robot2">
       <include file="$(find pioneer3at)/launch/explorer_robot2.launch"/>
    </group>



</launch>
